regression/contracts/quantifiers-nested-01/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+Verification:
+This test asserts the postconditions of f1.
```

Comment:
Could we update the descriptions of these test cases to indicate exactly what kind of nesting we are checking for? For instance, this test case is checking a forall inside a forall. Other tests check for forall inside conjunction / disjunction etc.